index.html

```diff
@@ -29,48 +29,52 @@ <h2>Telemetry portal</h2>
             <div class="col-md-4">
                 <h3><i class="fa fa-bar-chart" aria-hidden="true"></i> Main tools</h3>
                 <div class="dashboard-list list-group" style="margin: 0">
+                    <a class="list-group-item" href="https://data.firefox.com/">
+                        Firefox Public Data Report
+                        <div class="dashboard-description">See the activity, behavior, and hardware configuration of Firefox users.</div>
+                    </a>
                     <a class="list-group-item" href="new-pipeline/dist.html">
                         Measurement Dashboard
                         <div class="dashboard-description">View the measure distributions and evolutions over time. <i>a.k.a</i> TMO</div>
                     </a>
+                    <a class="list-group-item" href="https://missioncontrol.telemetry.mozilla.org/">
+                      Mission Control <div class="dashboard-description">View key Firefox quality measures in near-realtime.</div>
```

Comment:
```suggestion
                      Mission Control <div class="dashboard-description">View key Firefox quality measures in near real-time.</div>
```

Since we're moving this line anyway, let's fix this to use what looks to be the prevailing punctuation for "near real-time"